templates/index.html

```diff
@@ -6,514 +6,677 @@
+    <div class="row--25-75">
+      <div class="col u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/080a201e-intel-logo.png",
+          alt="intel logo",
+          width="852",
+          height="312",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-6">
+            <p>
+              <i>"Canonical's inclusion of the Intel Visual Cloud Accelerator Card - Render as part of their newly launched Anbox Cloud solution will enable the delivery of enhanced cloud and mobile gaming experiences on Android devices, supporting an emerging industry opportunity today, and for the upcoming 5G era."</i>
+            </p>
+          </div>
+          <div class="col-3">
+            <p class="u-no-margin--bottom">
+              <strong>Lynn Comp</strong>
+            </p>
+            <p class="u-text--muted">Vice President, Data Platforms Group and General manager of the Visual Cloud Division, Intel</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="p-section" id="run-android">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <h2 class="p-section--shallow">Run Android in the cloud with Anbox Cloud</h2>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <table class="p-table--mobile-card" data-heading="Table comparing deployments to run Android in the cloud with Anbox Cloud">
+        <thead>
+          <tr>
+            <th></th>
+            <th>Anbox Cloud Appliance</th>
+            <th>Charmed Anbox Cloud</th>
+            <th>Fully managed deployment</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th><strong>Ultra low latency streaming</strong></th>
+            <td data-heading="Appliance"><i class="p-icon--success-grey">Yes</i></td>
+            <td data-heading="Regular deployment"><i class="p-icon--success-grey">Yes</i></td>
+            <td data-heading="Fully managed deployment"><i class="p-icon--success-grey">Yes</i></td>
+          </tr>
+          <tr>
+            <th><strong>GPU Support</strong></th>
+            <td data-heading="Appliance">NVIDIA, AMD, Intel</td>
+            <td data-heading="Regular deployment">NVIDIA, AMD, Intel</td>
+            <td data-heading="Fully managed deployment">NVIDIA, AMD, Intel</td>
+          </tr>
+          <tr>
+            <th><strong>UI</strong></th>
+            <td data-heading="Appliance"><i class="p-icon--success-grey">Yes</i></td>
+            <td data-heading="Regular deployment"><i class="p-icon--success-grey">Yes</i></td>
+            <td data-heading="Fully managed deployment"><i class="p-icon--success-grey">Yes</i></td>
+          </tr>
+          <tr>
+            <th><strong>Android versions</strong></th>
+            <td data-heading="Appliance">10, 11, 12, 13</td>
```

Comment:
It should be 12, 13 and 14 as covered in the copy doc.

Reply:
Thanks, updated 👍